Standards/CCSMB-5.md

```diff
@@ -0,0 +1,23 @@
+# CCSMB-5: Radio Transmissions
+*Author: Emma Beurskens <@emmaknijn>*
+*Version: v1.0.0*
+*Last updated: YYYY-MM-DD*
+
+## Design
+A client will only listen to packets while the audio is being transmitted from a server
+The server will communicate to the client over modem
+
+## Packet
+A packet consists of the following fields in a table:
+- buffer: A table that contains Signed 8-bit PCM @ 48kHz audio
+- station: A string that contains the name of the radio station
+- title: A string that contains the song that is currently being played
+- protocol: A string that contains the protocol used
+
+In order to be compliant the
+- protocol field must be "CCSMB-5"
+- client must be able to accept the buffer field
+- client must be able to deal with crosstalk by allowing the user to select a station if there are multiple stations on a single channel
```

Comment:
What happens if two servers advertise the same "station" on the same channel?

Reply:
In short: chaos
With the id parameter it isnt that bad but bad actors still can cause chaos
Basically the client will try to play both servers so audio will be garbled